
Well, the usual outside kernel use madness and the new input-utils already utilize
a an ID define that did not yet hit mainline.

  - Rene Rebe <rene@exactcode.de>

--- usr/include/linux/serio.h	2005-04-29 23:37:56.000000000 +0200
+++ usr/include/linux/serio.h	2005-05-21 21:31:31.000000000 +0200
@@ -10,12 +10,12 @@
  */
 
 #include <linux/ioctl.h>
-#include <linux/interrupt.h>
 
 #define SPIOCSTYPE	_IOW('q', 0x01, unsigned long)
 
 #ifdef __KERNEL__
 
+#include <linux/interrupt.h>
 #include <linux/list.h>
 #include <linux/spinlock.h>
 #include <linux/device.h>
@@ -181,6 +181,7 @@
 #define SERIO_SNES232	0x26
 #define SERIO_SEMTECH	0x27
 #define SERIO_LKKBD	0x28
+#define SERIO_ELO	0x29
 
 #define SERIO_ID	0xff00UL
 #define SERIO_EXTRA	0xff0000UL
